homeassistant/components/aqualogic/sensor.py

```diff
@@ -91,12 +149,13 @@ def async_update_callback(self):
         """Update callback."""
         panel = self._processor.panel
         if panel is not None:
-            if panel.is_metric:
-                self._attr_native_unit_of_measurement = SENSOR_TYPES[self._type][1][0]
-            else:
-                self._attr_native_unit_of_measurement = SENSOR_TYPES[self._type][1][1]
+            self._attr_native_unit_of_measurement = (
+                self.entity_description.unit_metric
+                if panel.is_metric
+                else self.entity_description.unit_imperial
+            )
```

Comment:
Use regular if/else when it spans over multiple lines

Reply:
I did this initially. However, since the names are quite long I think it's actually more readable with the inline if. Pushed a commit to update it nevertheless. Would be happy to revert it though if you agree with me here.

Reply:
Personally, I agree. But it does not conform with the rest of the codebase.